packages/frontend-2/components/workspace/ProjectList.vue

```diff
@@ -1,5 +1,13 @@
 <template>
   <div>
```

Comment:
I dont know if it's my local and/or related, but search seems broken?

<img width="736" alt="Screenshot 2024-12-18 at 17 03 28" src="https://github.com/user-attachments/assets/32205981-6518-4d60-bb84-6681cd7d2325" />

Reply:
<img width="893" alt="image" src="https://github.com/user-attachments/assets/fcaa4b8d-d312-420d-902e-b2bdb9fc747c" />
Seems to work ok here, can you try with latest and see if it's working again?